commands/pull_request.go

```diff
@@ -156,6 +159,16 @@ func pullRequest(cmd *Command, args *Args) {
 	fullBase := fmt.Sprintf("%s:%s", baseProject.Owner, base)
 	fullHead := fmt.Sprintf("%s:%s", headProject.Owner, head)
 
+	if web {
+		openPullRequestURL := fmt.Sprintf("https://github.com/%s/compare/%s...%s?expand=1", baseProject, base, fullHead)
```

Comment:
This URL has hardcoded "github.com". We support other GitHub Enterprise hosts as well, so it's not a good idea to hardcode a single host name. Instead, please use ```

```go
baseProject.WebURL("", "", fmt.Sprintf("compare/%s...%s?expand=1", base, fullHead))
```